app/code/core/Mage/Catalog/Model/Resource/Abstract.php

```diff
@@ -634,7 +634,7 @@ public function getAttributeRawValue($entityId, $attribute, $store)
         if ($staticAttributes) {
             $select = $adapter->select()->from($staticTable, $staticAttributes)
                 ->where($this->getEntityIdField() . ' = :entity_id');
-            $attributesData = $adapter->fetchRow($select, array('entity_id' => $entityId));
+            $attributesData = $adapter->fetchRow($select, array('entity_id' => $entityId)) ?: [];
```

Comment:
using old array syntax `array` and new `[]` on the same line looks weird

Reply:
[ads]But #1835 will spank you if you use old array syntax.[/ads]

Reply:
I'm not telling it is wrong or good, just that it looks weird with hope there may exists instructions which syntax to use :)